docs/nlu/using-only-nlu.rst

```diff
@@ -0,0 +1,54 @@
+:desc: Find out how to use only Rasa NLU as a standalone NLU service for your chatbot or virtual assistant.
+
+.. _using-only-nlu:
+
+Using NLU Only
+==============
+
+
+If you want to use Rasa only as an NLU component, you can!
+
+Training NLU-only models
+------------------------
+
+To train an NLU model only, run:
+
+.. code-block:: bash
+
+   rasa train nlu
+
+This will look for NLU training data files in the ``data/`` directory
+and saved a trained model in the ``models/`` directory. 
+The name of the model will start with ``nlu-``.
+
+Testing your NLU model on the command line
+------------------------------------------
+
+To try out your NLU model on the command line, use the ``rasa shell`` command,
+passing in the name of your model with the ``-m`` argument. For example:
+
+.. code-block:: bash
+
+    rasa shell -m models/nlu-20190515-144445.tar.gz
```

Comment:
```suggestion
    rasa shell nlu -m models/nlu-20190515-144445.tar.gz
```

Reply:
I tested this locally without the nlu arg and it also worked

Reply:
so, if you have a combined model, you have to pass the `nlu` arg to shell, otherwise you don't. is that right?

Reply:
If you have a combined model and executing `rasa shell` it always starts the bot. 
If you have a combined model and executing `rasa shell nlu` it will start the NLU interpreter.